tianshou/env/ma_env.py

```diff
@@ -0,0 +1,90 @@
+import gym
+import numpy as np
+from abc import ABC, abstractmethod
+from typing import Tuple, Optional
+
+
+class MAEnv(ABC, gym.Env):
```

Comment:
I don't like this class name. Not explicit enough.

Reply:
Fixed in 279fd28 .